intf_.mod

```diff
@@ -572,11 +571,14 @@ PROCEDURE record () {
   VERBATIM {
   int k; double ti;
   vp = SOP;
-  if (tg>=t) return;
-  if (vp->p >= vp->size) { if (errflag) return; 
+  if (tg>=t) return 0;
+  if (vp->p >= vp->size) {
+    if (errflag) return 0;
     printf("**** WARNING out of recording room for INTF type%d id%d at %g****\n",IDP->type,IDP->id,t);
     printf("**************** WARNING: No further WARNINGS ****************\n");
-    errflag=1; return; }
+    errflag=1;
+    return 0;
+  }
```

Comment:
`PROCEDURE` has `int` return type when translated to C/C++ code. Hence add `return 0` instead of `return`.